packages/web-twig/src/Resources/components/Breadcrumbs/README.md

```diff
@@ -88,19 +88,46 @@ Without lexer:
 | ------------- | -------- | ------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `elementType` | `string` | `nav`   | ✕        | HTML tag to render                                                                                                                                                                                                                                                                |
 | `goBackTitle` | `string` | —       | ✕        | Title/translation for back link to previous page on mobile. It's essential to be set along with items. If items property is not passed, backlink is to be created within children property. [**Optional DEPRECATED**][Deprecated] Will be **required** in the next major version. |
 | `items`       | `array`  | `[]`    | ✕        | Navigation menu items                                                                                                                                                                                                                                                             |
 
+## BreadcrumbsItem
+
+Use the `BreadcrumbsItem` component for the ordered list as the component's children instead of passing the breadcrumb items array via props:
+
+```twig
+<Breadcrumbs>
+  {% for item in items %}
+    <BreadcrumbsItem isCurrent={{ items.length === index - 1 }} href={{ item.url }}>
+        {{ item.title }}
+    </BreadcrumbsItem>
+  {% endfor %}
+</Breadcrumbs>
+```
+
+### API
+
+| Name               | Type            | Default         | Required | Description                                           |
+| ------------------ | --------------- | --------------- | -------- | ----------------------------------------------------- |
+| `children`         | `string`        | —               | ✕        | Custom content to override items rendering from array |
+| `href`             | `string`        | —               | ✔        | URL                                                   |
+| `isCurrent`        | `boolean`       | `false`         | ✕        | Whether is the item the current page                  |
+| `isGoBackOnly`     | `boolean`       | `fasle`         | ✕        | Whether should be displayed in go back mode           |
+| `iconNameEnd`      | `string`        | `chevron-right` | ✕        | Icon component on the end of the item wrapper         |
+| `iconNameStart`    | `string`        | -               | ✕        | Icon component on the start of the item               |
```

Comment:
```suggestion
| `iconNameStart`    | `string`        | `chevron-left`               | ✕        | Icon component on the start of the item               |
```

and maybe rewrite the description, because we change only the Icon component name? (same applies to the end icon)